docs/source/export.md

```diff
@@ -68,4 +68,4 @@ pai -name easy_rec_ext -project algo_public
 - -DossHost: ossHost地址
 - -Dbuckets: config所在的bucket和保存模型的bucket; 如果有多个bucket，逗号分割
 - 如果是pai内部版,则不需要指定arn和ossHost, arn和ossHost放在-Dbuckets里面
-  - -Dbuckets=oss://easyrec/?role_arn=acs:ram::xxx:role/ev-ext-test-oss&host=oss-cn-beijing-internal.aliyuncs.com
+  - -Dbuckets=oss://easyrec/?role_arn=acs:ram::xxx:role/aliyunodpspaidefaultrole&host=oss-cn-beijing-internal.aliyuncs.com
```

Comment:
![image](https://user-images.githubusercontent.com/58054/170848862-71a34cad-6786-4e50-bbc7-5813dd141e12.png)

here is the emoji effect of ":ram:"